Update capture.md

libbyhopfauf opened the pull request:
This update includes reorganizing the capture instructions to include a section for CLI vs. GUI. The CLI instructions were also broken down to include a section explaining how to build a CLI string, examples of frequently used capture commands and a list of the full commands (the last of which still needs to be updated and examples need to be added).

dericed commented on docs/_sections/capture.md:
````diff
@@ -3,4 +3,107 @@ layout: post
+
+7.) Type the files name. Add a space. Example: ```filename.dv```
+
+8.) Type ```-m``` to note you want to merge the multiple capture attempts for problematic frames (dvrescue will automatically rewind and try to capture the frame again when it encouters a problematic frame).
````
this `-m` also needs an output `-` in this case, so the total would be `-m -`

dericed commented on docs/_sections/capture.md:
````diff
@@ -3,4 +3,107 @@ layout: post
+
+2.) You can either type ```dvrescue``` at the start of the command or include the path to the version of dvrescue you want to use. Example: ```/usr/local/bin/dvrescue```
+
+4.) If you only have one device connected, dvrescue will automatically select it (device 0). If you have more than one device connected, add the device number ```device://1``` (for the corresponding deck you want to use for capture). Example (using the deck listed in step 1): ```dvrescue device://0``` 
````
hmmm, don't you have the full guid in your result such as
```
$ dvrescue --list_devices
90:a1838cb4:00000000: UltraStudio 4K Mini [DeckLink]
```

libbyhopfauf replied:
Whoops, looks like that was an old example. Will update. So people need to add full number if they have more than one device plugged in right?

libbyhopfauf replied:
@dericed See latest additions to the pull request for changes :)

dericed commented on docs/_sections/capture.md:
```diff
@@ -3,4 +3,107 @@ layout: post
+
+**--list_devices** = List detected devices and their indices.
+
+**--statusl** = Provide the status (playing, stop...) of the input.<br />
```
I think you're testing against a pretty old CLI, since the statusl typo was fixed long ago.

libbyhopfauf replied:
LOL my bad, this was actually copied out of the old documentation. My CLI is up to date. I just didn't have a deck to test haha.

libbyhopfauf replied:
I changed it to have the latest version of dvrescue -h

libbyhopfauf commented:
@dericed I made all of your suggested changes and made sure all of the latest updates are in there. But them me know if you catch anything else I should change before merging this in :)

libbyhopfauf commented:
@dericed can this be merged?

libbyhopfauf commented:
So, I'm hitting a little bit of a brick wall trying to make the CLI documentation accurate. These commands work: 

**Capture from a specific device:**
```dvrescue device://0xdevicename -m /pathtofile/filename.dv``` 
For example:
```dvrescue device://0x800460104ce2bad -m /Users/libbyshopfauf/Desktop/12345.dv```

**Capture from a specific device and show a preview window:** 
```dvrescue device://0xdevicename -m /pathtofile/filename.dv -m - | ffplay -``` 
For example:
```dvrescue device://0x800460104ce2bad -m /Users/libbyshopfauf/Desktop/12345.dv -m - | ffplay -```

These commands do not:
**Capture from a specific device using rewind, recapture and merge (you can only specify a number of times to rewind and recapture if you specify a device):**
```dvrescue device://0xdevicename --rewind-count # -m filename.dv -m -``` 
For example:
```dvrescue device://0x800460104ce2bad --rewind-count 3 -m /Users/libbyshopfauf/Desktop/12345.dv -m -```

**Capture from a specific device using rewind, recapture and merge (3 attempts), show a preview window (with only one deck connected):**
```dvrescue --rewind-count 3 -m filename.dv -m - | ffplay -```

**Basic capture (with only one deck connected):**
```dvrescue --rewind-count 3 -m filename.dv -m - | ffplay -```
</details>

libbyhopfauf commented:
@dericed I just saw #914 regarding rewind and merge. So that might explain why it seemed like rewind and recapture was working even when I didn't have it as part of the command.

libbyhopfauf commented:
@dericed this is ready to be merged! I updated the images to reflect how the GUI looks and what information is present for capture in the latest build. I also tested and updated all of the CLI commands and examples. Everything works and is good to go!
